app/utils/splitTitle.ts

```diff
@@ -0,0 +1,6 @@
+export default function splitTitle(title: string) {
+  const titleSplit = title.split("-");
+  const artist = titleSplit[0].trimEnd();
+  const album = titleSplit[1].trimStart();
+  return [artist, album];
```

Comment:
I'd return this as an object, that way when you call this you know what the return values actually are.

I'd only return a tuple if it made sense for the destructed variable names to be different (like `useState` for example since the destructured name we want is dependent on the context)

```suggestion
  return { artist, album };
```